code/datums/spells/lightning.dm

```diff
@@ -0,0 +1,89 @@
+/obj/effect/proc_holder/spell/targeted/lightning
+	name = "Lightning Bolt"
+	desc = "Throws a lightning bolt at the nearby enemy. Classic."
+	charge_type = "recharge"
+	charge_max	= 300
+	clothes_req = 1
+	invocation = "UN'LTD P'WAH!"
```

Comment:
@Dorsisdwarf How about using random wizard-ized (the apostrophes) swearwords?
https://www.youtube.com/watch?v=X_d4j3UlZ18